.golangci.yml

```diff
@@ -1,5 +1,4 @@
 run:
-  go: 1.19
```

Comment:
If we remove overriding the go version here, the default will be taken from the go.mod: https://golangci-lint.run/usage/configuration/#run-configuration. Effectively making it the single source of truth for the go version.